src/layout/TheSidebar.vue

```diff
@@ -0,0 +1,186 @@
+            <span
+              :title="tabName"
+              :class="[
+                'icon-after',
+                tabName === NotitieTabs.Terugbel ? 'phone-flip' : 'note',
+              ]"
+            ></span>
+          </template>
+          <template #[NotitieTabs.Regulier]>
+            <utrecht-heading id="notitieblok" model-value :level="2"
+              >Notitieblok</utrecht-heading
+            >
+            <textarea
+              aria-labelledby="notitieblok"
+              v-focus
+              class="utrecht-textarea"
+              v-model="contactmomentStore.huidigeVraag.notitie"
+            />
+          </template>
+          <template #[NotitieTabs.Terugbel]>
+            <p v-if="contactverzoekIsSentToMedewerker">
+              Contactverzoek verstuurd naar
+              {{ contactverzoekIsSentToMedewerker }}
+            </p>
+            <ContactverzoekFormulier
+              :huidige-vraag="contactmomentStore.huidigeVraag"
+              :huidige-klant="contactmomentStore.klantVoorHuidigeVraag"
+              @start="handleContactverzoekStart"
+              @submit="handleContactverzoekSubmit"
+            />
+          </template>
+        </tabs-component>
+      </section>
+    </template>
+  </aside>
+</template>
+
+<script lang="ts" setup>
+import { ContactverzoekFormulier } from "@/features/contactverzoek";
+import TabsComponent from "@/components/TabsComponent.vue";
+import { UtrechtHeading } from "@utrecht/web-component-library-vue";
+import ContactmomentVragenMenu from "@/features/contactmoment/ContactmomentVragenMenu.vue";
+import { useContactmomentStore } from "@/stores/contactmoment";
+import { ensureState } from "@/stores/create-store";
+import { watch, computed } from "vue";
+import { useRoute } from "vue-router";
+import ContactmomentStarter from "../features/contactmoment/ContactmomentStarter.vue";
+
+enum NotitieTabs {
+  Regulier = "Reguliere notitie",
+  Terugbel = "Contactverzoek",
+}
+
+const contactmomentStore = useContactmomentStore();
+const route = useRoute();
+
+const state = ensureState({
+  stateId: "Sidebar",
+  stateFactory() {
+    return {
+      currentNotitieTab: NotitieTabs.Regulier,
+    };
+  },
+});
+
+watch(
+  () => contactmomentStore.huidigeVraag,
+  () => {
+    state.reset();
+  }
+);
+
+const contactverzoekIsSentToMedewerker = computed(
+  () => contactmomentStore.huidigeVraag.contactverzoek.medewerker
+);
+
+const handleContactverzoekStart = () => {
+  contactmomentStore.huidigeVraag.contactverzoek.isInProgress = true;
+};
+
+const handleContactverzoekSubmit = ({
+  medewerker,
+  url,
+}: {
+  medewerker: string;
+  url: string;
+}) => {
+  const { contactverzoek } = contactmomentStore.huidigeVraag;
+  contactverzoek.url = url;
+  contactverzoek.medewerker = medewerker;
+  contactverzoek.isSubmitted = true;
+  contactverzoek.isInProgress = false;
+};
+</script>
+
+<style lang="scss" scoped>
+aside {
+  --color-dark: #352b75;
+  --color-light: #c0c4fc;
```

Comment:
I think, when introducing new colors, it's best to do this on a global level. To (1) have the ability to reuse them in other places and (2) have a clear overview of all colors used in the project "om wildgroei te voorkomen"

Reply:
I agree. I share your doubts about the color pallet so I didn't want to encourage re-use for now. But that doesn't make it more clear. I will put them at the global level and add a comment.